docs/examples/README.md

```diff
@@ -0,0 +1,60 @@
+# Get started with examples
+
+Learn how to install Playwright, set up your dev environment to author Playwright scripts, and example recipes to bootstrap your scripts.
+
+## Installing Playwright
+
+Playwright is a Node.js library and can be acquired through the npm registry. Use npm or yarn to install Playwright in your Node.js project.
+
+```
+npm i playwright
+```
+
+Once installed, you can `require` Playwright in your Node.js scripts. 
+
+```js
+const playwright = require('playwright');
+
+(async () => {
+  const browser = await playwright.chromium.launch();
+  await browser.close();
+})();
+```
+
+## Setup dev environment
+
+Playwright scripts can be developed just like any other Node.js script. You can use the [Node.js debugger](https://nodejs.org/api/debugger.html) or [VS Code debugging](https://code.visualstudio.com/docs/nodejs/nodejs-debugging) to set breakpoints and get fine grained control over execution.
+
+### Running browsers for debugging
+
+By default, Playwright runs the browsers in headless mode. To see the browser UI, pass the `headless: false` flag while launching the browser. You can also use `slowMo` to slow down execution.
+
+```js
+  playwright.firefox.launch({ headless: false, slowMo: 50 });
```

Comment:
- `chromium.launch(...)`
- Let's use chromium everywhere for consistency, and mention at the top that all examples use Chromium, but can be similarly run with Firefox/WebKit?